### PR TITLE
update solid info

### PR DESCRIPTION
**Pull request by chiefcll**

### CODE REVIEW

File: site/.vitepress/theme/components/home/HeroDisplay.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@ import LinkButton from './LinkButton.vue';
         <div class="mx-auto max-w-2xl pt-20 sm:pt-28">
         <div class="hidden sm:mb-8 sm:flex sm:justify-center">
             <div class="relative rounded-full px-3 py-1 text-sm leading-6 text-gray-900 dark:text-gray-200 ring-1 ring-gray-900/10 hover:ring-gray-900/20">
-                Lightningjs 3.0 <a href="https://lightningjs.io/v3-docs/blits/getting_started/intro.html" target="_blank" class="font-semibold text-indigo-600"><span class="absolute inset-0" aria-hidden="true" />Read more <span aria-hidden="true">&rarr;</span></a>
+                Lightningjs 3.0 <a href="https://github.com/lightning-js/renderer?tab=readme-ov-file#readme" target="_blank" class="font-semibold text-indigo-600"><span class="absolute inset-0" aria-hidden="true" />Read more <span aria-hidden="true">&rarr;</span></a>
```
Comment (wouterlucas) [inline site/.vitepress/theme/components/home/HeroDisplay.vue:10]: Pointing folks to the renderer directly will create confusion, please leave this as it was.

File: site/.vitepress/theme/components/home/HeroDisplay.vue [hunk 2]
```diff
@@ -18,8 +18,12 @@ import LinkButton from './LinkButton.vue';
...
+            <p class="mt-6 text-lg leading-8 text-black dark:text-gray-200">Lightweight, open-source framework for developing hyper fast animated applications. Get started with:</p>
+            <div class="flex items-center justify-center gap-10 py-5">
+                <a href="https://lightningjs.io/v3-docs/blits/intro.html" target="_blank" class="transition ease-out duration-300 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:-translate-y-1 hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"> Blits <span aria-hidden="true">→</span></a>
+                or
+                <a href="https://lightning-js.github.io/solid/intro.html" target="_blank" class="transition ease-out duration-300 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:-translate-y-1 hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"> SolidJS <span aria-hidden="true">→</span></a>
+            </div>
```
Comment (wouterlucas) [inline site/.vitepress/theme/components/home/HeroDisplay.vue:26]: We should think about the UX here, in the future React and maybe others will join. I think we shouldn't fragment the site with too much and keep the alternatives in 1 separate place.

File: site/.vitepress/theme/layouts/HomeLayout.vue [hunk 3]
```diff
@@ -83,14 +83,18 @@ const home = {
...
+        description: 'Simple and performant reactivity for building user interfaces with Lightning or the Web.',
         logo: img_solid,
         darkLogo: img_solid_dark,
         code: solid_code,
         codeType: 'jsx',
         points: [
-          'JSX',
-          'Solidjs Reactivity and Functionalities'
+          'Familiar JSX Syntax + TypeScript',
+          'Develop for Lightning and Web',
```
Comment (wouterlucas) [inline site/.vitepress/theme/layouts/HomeLayout.vue:94]: please remove web, that goes directly into what Lightning 3 is trying to do. Lets try to not confuse our users.

File: site/v3-docs/solid/intro.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<p>
+  <img src="https://assets.solidjs.com/banner?project=Library&type=solid-lightning" alt="SolidJS Lightning" />
+</p>
+
+# SolidJS for Lightning 3
+
+Solid-Lightning is a UI framework for the [Lightning 3 Renderer](https://github.com/lightning-js/renderer) built with [SolidJS](https://www.solidjs.com/) [Universal Renderer](https://github.com/solidjs/solid/releases/tag/v1.2.0). It allows you to declaratively construct lightning nodes with reactive primitives, just as you would construct a DOM tree in SolidJS.
```
Comment (wouterlucas) [inline site/v3-docs/solid/intro.md:7]: I dont think that last part is true, you're not constructing "just as you would construct a DOM tree". The dom primitives won't work when creating a SolidJS lightning app.

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: site/.vitepress/theme/components/home/HeroDisplay.vue (hunk 2 above)
Comment (chiefcll) [reply site/.vitepress/theme/components/home/HeroDisplay.vue:26]: Will React + React Native be official projects? Are they part of Lightning? We should think about the UX, we have two projects and only one is being pointed to.

File: site/.vitepress/theme/components/home/HeroDisplay.vue (hunk 1 above)
Comment (chiefcll) [reply site/.vitepress/theme/components/home/HeroDisplay.vue:10]: Is Lightning 3 the renderer or is it Blits? There should be a new Readme overview detailing

File: site/.vitepress/theme/layouts/HomeLayout.vue (hunk 3 above)
Comment (chiefcll) [reply site/.vitepress/theme/layouts/HomeLayout.vue:94]: I'll reword this

File: site/v3-docs/solid/intro.md (hunk 4 above)
Comment (chiefcll) [reply site/v3-docs/solid/intro.md:7]: 👍

**Comment by chiefcll** [comment]
Just so folks don't have to run it - here is what this looks like:

image

**Comment by chiefcll** [comment]
@wouterlucas - comments are addressed can we get the website fixed up?

**Pull request closed without merging**